content/blog/2021/compact_filters_demo.md

```diff
@@ -0,0 +1,367 @@
+  ```
+  (Also check other information returned by `bdk-cli create_tx`)  
+
+### Sign and Broadcast the transaction
+Asking BDK to sign a transaction is as straight forward as it can get. BDK already holds the `xprv` deatils to sign a transaction. It returns a finalised `signed_psbt` which we will next broadcast to the network.
+	
+- Sign the transaction
+  ```shell
+  $ signed_psbt=$(bdk-cli --network regtest wallet --node "127.0.0.1:18444" --wallet bdk-test -d $BDK_recv_desc -c $BDK_chng_desc sign --psbt $psbt | jq -r '.psbt')
+  ```  
+
+- Broadcast the transaction
+  ```shell
+  $ bdk-cli --network regtest wallet --node "127.0.0.1:18444" --wallet bdk-test -d $BDK_recv_desc -c $BDK_chng_desc broadcast --psbt $signed_psbt 
+  {
+    "txid": "c343f5b25372e285308eba912d1fe8fade9f64afde6d95306e248e52e0852252"
+  }
+  ```
+  This makes BDK broadcast the transaction via the connected core node, and it returns the corresponding Txid.
+
+### Confirming the Transaction
+ The transaction has been received by the `core` node and waiting in its mempool for inclusion in block.
+ We can see the transaction via its `txid` received in previous step.
+
+- Check transaction in mempool
+  ```shell
+  $ bitcoin-cli gettransaction c343f5b25372e285308eba912d1fe8fade9f64afde6d95306e2248e52e0852252
+  {
+    "amount": 2.00000000,
+    "confirmations": 0,
+    "trusted": false,
+    "txid": "c343f5b25372e285308eba912d1fe8fade9f64afde6d95306e248e52e0852252",
+    "walletconflicts": [
+    ],
+    "time": 1621697202,
+    "timereceived": 1621697202,
+    "bip125-replaceable": "no",
+    "details": [
+      {
+        "address": "bcrt1q3h4hs6mve5dcl7da3d4acmlp20hh8c3t4mldwe",
+        "category": "receive",
+        "amount": 2.00000000,
+        "label": "",
+        "vout": 1
+      }
+    ],
+    "hex": "01000000000101d84e8cb7477f9fe6f265b56d5416ff47da9a70be18f65ec50731b8257c67f2bd0100000000ffffffff0273a2e11100000000160014874270187001febc4cebd8cb083cf2c783e8f1ac00c2eb0b000000001600148deb786b6ccd1b8ff9bd8b6bdc6fe153ef73e22b0247304402201037d9ef5b80392296311c8899b1f12a0987778d694a442a88bafa6fbd7a7c9a022011293176255897444d9c71b0b9cd13b2aedb749b142577566c90a63d61025e2c01210202427d16b29c1c8546255363a74326ee9ab3196770bb3fccc7b679d52f9c1ccf00000000"
+  }
+  ```
+  This means, core has recieved the transaction in its mempool and waiting for confirmation.
+
+- Generate 1 block to confirm the transaction
+  ```shell
+  $ bitcoin-cli generatetoaddress 1 bcrt1qatd7yq0jukwusuaufltlejmeydpvnpv43r5gc2
+  [
+    "55436ff0169bbb3e70ab10cb7cdd45ab86204d5d7864a109142d91120d023197"
+  ]
+  ```
+
+- Sync the `bdk-test` wallet and ask for available balance.
+  ```shell
+	$ bdk-cli --network regtest wallet --node 127.0.0.1:18444 --wallet bdk-test --descriptor $desc sync
+	{}
+	
+	$ bdk-cli --network regtest wallet --node 127.0.0.1:18444 --wallet bdk-test --descriptor $desc get_balance
+	{
+		"satoshi": 299999859
+	}
+  ```
+
+	Voila! We have the updated balance. 
+
```

Comment:
I would also add a step to kill the container, so it will get cleaned up in case they want to run the tutorial again, something like: 

```suggestion
You may now shutdown the regtest docker container, this will also clean up any created data.

  $ docker kill bdk-box

```

Reply:
Ya makes sense..